### PR TITLE
Donut migration

### PR DESCRIPTION
**Pull request by Moonyman**
- Filled the recent donut void in the sec vendor with 5 peppersprays
- Accomodated evicted from sec vendor donuts in the coffee vendor

### CODE REVIEW

File: code/game/machinery/vending_types.dm [hunk 1]
```diff
@@ -359,9 +362,10 @@
 		/obj/item/weapon/grenade/flashbang = 4
 	)
 	restock_blocked_items = list(
-		/obj/item/weapon/storage/box/donut, 
+		/obj/item/weapon/storage/box/donut,
```
Comment (Arrow768) [inline code/game/machinery/vending_types.dm]: Just readd the donut boxes to the security vendor.
It seems I just fogot to add them to the products list and only added them to the restock_blocked_items

Comment (Moonyman) [reply code/game/machinery/vending_types.dm]: donuts boxes are still there in the premium list as before (i presume that's what coins get you)

File: code/game/machinery/vending_types.dm [hunk 2]
```diff
@@ -108,15 +108,17 @@
...
 	)
 	contraband = list(
 		/obj/item/weapon/reagent_containers/food/drinks/ice = 10
 	)
 	prices = list(
 		/obj/item/weapon/reagent_containers/food/drinks/coffee = 20,
 		/obj/item/weapon/reagent_containers/food/drinks/tea = 20,
-		/obj/item/weapon/reagent_containers/food/drinks/h_chocolate = 22
+		/obj/item/weapon/reagent_containers/food/drinks/h_chocolate = 22,
+		/obj/item/weapon/reagent_containers/food/snacks/donut/normal = 18
```
Comment (Sindorman) [inline code/game/machinery/vending_types.dm]: Uhm.. just a suggestion. Maybe price is too high? The coffee is usually around 1.21$ in US and donuts could be as cheap as 69 cents. So donuts are 1.75 times less of a price then coffee. Here they are 0.1 times less, which is very close.

Comment (Moonyman) [reply code/game/machinery/vending_types.dm]: I'll fix it. It's kinda hard to get a firm grasp on pricing without persistent economy

Comment (Sindorman) [reply code/game/machinery/vending_types.dm]: That is very true

**Pull request merged**